2019-03-29 19:50:31,113 : INFO : Machine : Mission Pinball Framework Core Engine v0.52.1
2019-03-29 19:50:31,114 : INFO : Machine : Command line arguments: {'no_load_cache': False, 'create_config_cache': True, 'bcp': True, 'configfile': ['config.yaml'], 'mpfconfigfile': 'c:\\users\\wilder\\appdata\\local\\programs\\python\\python36\\lib\\site-packages\\mpf\\mpfconfig.yaml', 'force_assets_load': False, 'jsonlogging': False, 'logfile': 'logs\\2019-03-29-19-50-31-mpf-RedTunic.log', 'pause': False, 'production': False, 'text_ui': True, 'loglevel': 10, 'consoleloglevel': 20, 'force_platform': None, 'syslog_address': None, 'mc_file_name': None, 'no_sound': False}
2019-03-29 19:50:31,114 : INFO : Machine : MPF path: c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\mpf
2019-03-29 19:50:31,114 : INFO : Machine : Machine path: C:\Users\Wilder\zelda
2019-03-29 19:50:31,114 : INFO : Machine : Platform: win32
2019-03-29 19:50:31,114 : INFO : Machine : Python executable location: c:\users\wilder\appdata\local\programs\python\python36\python.exe
2019-03-29 19:50:31,114 : INFO : Machine : Python version: 3.6.4 (64-bit)
2019-03-29 19:50:31,115 : INFO : Machine : Machine config file #1: config.yaml
2019-03-29 19:50:31,117 : INFO : ConfigProcessor : Loading config from cache: C:\Users\Wilder\AppData\Local\Temp\9bf880992bc5b85aa1bd2cc37b214e56.mpf_cache
2019-03-29 19:50:31,119 : DEBUG : asyncio : Using selector: SelectSelector
2019-03-29 19:50:31,120 : INFO : Machine : Initialise MPF.
2019-03-29 19:50:31,172 : DEBUG : FAST : Configuring FAST hardware.
2019-03-29 19:50:31,379 : INFO : FAST : Connecting to \\.\COM11 at 921600bps
2019-03-29 19:50:31,423 : INFO : FAST : Connected! Processor: NET, Board Type: FP-CPU-002-2, Firmware: 01.02
2019-03-29 19:50:31,423 : INFO : EventManager : Event: ======'machine_var_fast_net_firmware'====== Args={'value': '01.02', 'prev_value': None, 'change': True}
2019-03-29 19:50:31,423 : INFO : EventManager : Event: ======'machine_var_fast_net_model'====== Args={'value': 'FP-CPU-002-2', 'prev_value': None, 'change': True}
2019-03-29 19:50:31,423 : WARNING : FAST : Not resetting FAST NET because firmware is currently broken.
2019-03-29 19:50:31,487 : INFO : FAST : Connecting to \\.\COM12 at 921600bps
2019-03-29 19:50:31,535 : INFO : FAST : Connected! Processor: RGB, Board Type: FP-CPU-002-2, Firmware: 01.00
2019-03-29 19:50:31,535 : INFO : EventManager : Event: ======'machine_var_fast_rgb_firmware'====== Args={'value': '01.00', 'prev_value': None, 'change': True}
2019-03-29 19:50:31,535 : INFO : EventManager : Event: ======'machine_var_fast_rgb_model'====== Args={'value': 'FP-CPU-002-2', 'prev_value': None, 'change': True}
2019-03-29 19:50:31,536 : INFO : EventManager : Event: ======'machine_var_credits_string'====== Args={'value': 'FREE PLAY', 'prev_value': None, 'change': True}
2019-03-29 19:50:31,599 : DEBUG : YAML File Interface : Loading file: C:\Users\Wilder\zelda\data/machine_vars.yaml
2019-03-29 19:50:31,602 : INFO : EventManager : Event: ======'machine_var_player1_score'====== Args={'value': 64, 'prev_value': None, 'change': True}
2019-03-29 19:50:31,602 : INFO : EventManager : Event: ======'machine_var_mpf_version'====== Args={'value': 'MPF v0.52.1', 'prev_value': None, 'change': True}
2019-03-29 19:50:31,602 : INFO : EventManager : Event: ======'machine_var_mpf_extended_version'====== Args={'value': 'MPF v0.52.1, Config version:5, Show version: 5, BCP version:1.1', 'prev_value': None, 'change': True}
2019-03-29 19:50:31,602 : INFO : EventManager : Event: ======'machine_var_python_version'====== Args={'value': '3.6.4', 'prev_value': None, 'change': True}
2019-03-29 19:50:31,603 : INFO : EventManager : Event: ======'machine_var_platform'====== Args={'value': 'Windows-7-6.1.7601-SP1', 'prev_value': None, 'change': True}
2019-03-29 19:50:31,603 : INFO : EventManager : Event: ======'machine_var_platform_system'====== Args={'value': 'Windows', 'prev_value': None, 'change': True}
2019-03-29 19:50:31,603 : INFO : EventManager : Event: ======'machine_var_platform_release'====== Args={'value': '7', 'prev_value': None, 'change': True}
2019-03-29 19:50:31,603 : INFO : EventManager : Event: ======'machine_var_platform_version'====== Args={'value': '6.1.7601', 'prev_value': None, 'change': True}
2019-03-29 19:50:31,603 : INFO : EventManager : Event: ======'machine_var_platform_machine'====== Args={'value': 'AMD64', 'prev_value': None, 'change': True}
2019-03-29 19:50:31,603 : INFO : EventManager : Event: ======'init_phase_1'====== Args={}
2019-03-29 19:50:31,662 : INFO : ConfigProcessor : Loading config from cache: C:\Users\Wilder\AppData\Local\Temp\da675f6b180b79eecb40cb5eb7f75a18.mpf_cache
2019-03-29 19:50:31,666 : DEBUG : ModeController : Loaded mode bomb_finder
2019-03-29 19:50:31,666 : INFO : ConfigProcessor : Loading config from cache: C:\Users\Wilder\AppData\Local\Temp\43a06f52d0e9cf0597140b4c27159d40.mpf_cache
2019-03-29 19:50:31,669 : DEBUG : ModeController : Loaded mode escapehurryup
2019-03-29 19:50:31,670 : INFO : ConfigProcessor : Loading config from cache: C:\Users\Wilder\AppData\Local\Temp\8b601767b8d464abeaa513b42e6346a8.mpf_cache
2019-03-29 19:50:31,673 : DEBUG : ModeController : Loaded mode master_sword_buildup
2019-03-29 19:50:31,673 : INFO : ConfigProcessor : Loading config from cache: C:\Users\Wilder\AppData\Local\Temp\c35ad2e595070cac0b6f696d6f7f6af6.mpf_cache
2019-03-29 19:50:31,677 : DEBUG : ModeController : Loaded mode ball_save
2019-03-29 19:50:31,677 : INFO : ConfigProcessor : Loading config from cache: C:\Users\Wilder\AppData\Local\Temp\1f8ff34a8859f8c62d77912f33e29b9c.mpf_cache
2019-03-29 19:50:31,681 : DEBUG : ModeController : Loaded mode 2ballqualifier
2019-03-29 19:50:31,681 : INFO : ConfigProcessor : Loading config from cache: C:\Users\Wilder\AppData\Local\Temp\c3fd2c292a6c0d3463ad9b4555730ad5.mpf_cache
2019-03-29 19:50:31,685 : DEBUG : ModeController : Loaded mode item_tracker
2019-03-29 19:50:31,685 : WARNING : ConfigProcessor : Config file in cache changed: C:\Users\Wilder\zelda\modes\base\config\base.yaml
2019-03-29 19:50:31,685 : INFO : ConfigProcessor : Loading config from file C:\Users\Wilder\zelda\modes\base\config\base.yaml.
2019-03-29 19:50:31,686 : DEBUG : YAML File Interface : Loading file: C:\Users\Wilder\zelda\modes\base\config\base.yaml
2019-03-29 19:50:31,728 : INFO : ConfigProcessor : Loading config: C:\Users\Wilder\zelda\modes\base\config\base.yaml
2019-03-29 19:50:31,729 : INFO : ConfigProcessor : Config file cache created: C:\Users\Wilder\AppData\Local\Temp\4afc9f23ca6340163a8bb51d7deea78f.mpf_cache
2019-03-29 19:50:31,733 : DEBUG : ModeController : Loaded mode base
2019-03-29 19:50:31,733 : INFO : ConfigProcessor : Loading config from cache: C:\Users\Wilder\AppData\Local\Temp\58a9cd5447353eacedab412f1c10b0ab.mpf_cache
2019-03-29 19:50:31,738 : DEBUG : ModeController : Loaded mode lost_woods
2019-03-29 19:50:31,738 : INFO : ConfigProcessor : Loading config from cache: C:\Users\Wilder\AppData\Local\Temp\6fd2b5e36bbbd9c12f9ee3aa32f1ceca.mpf_cache
2019-03-29 19:50:31,742 : DEBUG : ModeController : Loaded mode ball_hold_tracker
2019-03-29 19:50:31,742 : INFO : ConfigProcessor : Loading config from cache: C:\Users\Wilder\AppData\Local\Temp\28e1731e516595966b3afcb08910cb63.mpf_cache
2019-03-29 19:50:31,748 : DEBUG : ModeController : Loaded mode heart_tracker
2019-03-29 19:50:31,748 : INFO : ConfigProcessor : Loading config from cache: C:\Users\Wilder\AppData\Local\Temp\dd39e2de39eb354a1ec99be2f3167765.mpf_cache
2019-03-29 19:50:31,753 : DEBUG : ModeController : Loaded mode heartmeter
2019-03-29 19:50:31,753 : INFO : ConfigProcessor : Loading config from cache: C:\Users\Wilder\AppData\Local\Temp\58664c35241624b07b13025a5964d2a4.mpf_cache
2019-03-29 19:50:31,759 : DEBUG : ModeController : Loaded mode video_player
2019-03-29 19:50:31,759 : INFO : ConfigProcessor : Loading config from cache: C:\Users\Wilder\AppData\Local\Temp\fd6423453738b6cee138971698f65579.mpf_cache
2019-03-29 19:50:31,764 : DEBUG : ModeController : Loaded mode mode_tracker
2019-03-29 19:50:31,764 : WARNING : ConfigProcessor : Config file in cache changed: C:\Users\Wilder\zelda\modes\playfield_multiplier\config\playfield_multiplier.yaml
2019-03-29 19:50:31,765 : INFO : ConfigProcessor : Loading config from file C:\Users\Wilder\zelda\modes\playfield_multiplier\config\playfield_multiplier.yaml.
2019-03-29 19:50:31,765 : DEBUG : YAML File Interface : Loading file: C:\Users\Wilder\zelda\modes\playfield_multiplier\config\playfield_multiplier.yaml
2019-03-29 19:50:31,773 : INFO : ConfigProcessor : Loading config: C:\Users\Wilder\zelda\modes\playfield_multiplier\config\playfield_multiplier.yaml
2019-03-29 19:50:31,774 : ERROR : asyncio : Exception in callback EventManager._async_handler_done(<QueuedEvent>)(<Task finishe...fig files.',)>) at c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\mpf\core\events.py:96
handle: <Handle EventManager._async_handler_done(<QueuedEvent>)(<Task finishe...fig files.',)>) at c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\mpf\core\events.py:96>
Traceback (most recent call last):
  File "c:\users\wilder\appdata\local\programs\python\python36\lib\asyncio\events.py", line 145, in _run
    self._callback(*self._args)
  File "c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\mpf\core\events.py", line 99, in _async_handler_done
    future.result()
  File "c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\mpf\core\mode_controller.py", line 117, in load_modes
    self.machine.modes[mode] = self._load_mode(mode)
  File "c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\mpf\core\mode_controller.py", line 266, in _load_mode
    config = self._load_mode_config(mode_string)
  File "c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\mpf\core\mode_controller.py", line 184, in _load_mode_config
    store_to_cache=self.machine.options['create_config_cache'])
  File "c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\mpf\core\config_processor.py", line 112, in load_config_files_with_cache
    ignore_unknown_sections)
  File "c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\mpf\core\config_processor.py", line 149, in _load_config_file_and_return_loaded_files
    'files.'.format(k, filename, config_type))
ValueError: Found a "player_vars:" section in config file C:\Users\Wilder\zelda\modes\playfield_multiplier\config\playfield_multiplier.yaml, but that section is not valid in mode config files.
2019-03-29 19:50:31,778 : INFO : Machine : Starting the main run loop.
2019-03-29 19:50:31,778 : INFO : Machine : Shutting down...
2019-03-29 19:50:31,778 : INFO : EventManager : Event: ======'shutdown'====== Args={}
2019-03-29 19:50:31,782 : ERROR : FAST : Stop called on serial connection \\.\COM12
2019-03-29 19:50:31,782 : ERROR : FAST : Stop called on serial connection \\.\COM11
2019-03-29 19:50:32,085 : ERROR : root : Found a "player_vars:" section in config file C:\Users\Wilder\zelda\modes\playfield_multiplier\config\playfield_multiplier.yaml, but that section is not valid in mode config files.
Traceback (most recent call last):
  File "c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\mpf\commands\game.py", line 220, in __init__
    MachineController(mpf_path, machine_path, vars(self.args)).run()
  File "c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\mpf\core\machine.py", line 709, in run
    self._run_loop()
  File "c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\mpf\core\machine.py", line 769, in _run_loop
    raise self._exception['exception']
  File "c:\users\wilder\appdata\local\programs\python\python36\lib\asyncio\events.py", line 145, in _run
    self._callback(*self._args)
  File "c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\mpf\core\events.py", line 99, in _async_handler_done
    future.result()
  File "c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\mpf\core\mode_controller.py", line 117, in load_modes
    self.machine.modes[mode] = self._load_mode(mode)
  File "c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\mpf\core\mode_controller.py", line 266, in _load_mode
    config = self._load_mode_config(mode_string)
  File "c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\mpf\core\mode_controller.py", line 184, in _load_mode_config
    store_to_cache=self.machine.options['create_config_cache'])
  File "c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\mpf\core\config_processor.py", line 112, in load_config_files_with_cache
    ignore_unknown_sections)
  File "c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\mpf\core\config_processor.py", line 149, in _load_config_file_and_return_loaded_files
    'files.'.format(k, filename, config_type))
ValueError: Found a "player_vars:" section in config file C:\Users\Wilder\zelda\modes\playfield_multiplier\config\playfield_multiplier.yaml, but that section is not valid in mode config files.
